todo_app/flask_config.py

```diff
@@ -1,9 +1,16 @@
-import os
-
-
-class Config:
-    def __init__(self):
-        """Base configuration variables."""
-        self.SECRET_KEY = os.environ.get('SECRET_KEY')
-        if not self.SECRET_KEY:
-            raise ValueError("No SECRET_KEY set for Flask application. Did you follow the setup instructions?")
+import os
+
+
+class Config:
+    TRELLO_API_KEY = os.environ.get("TRELLO_API_KEY")
+    TRELLO_API_SECRET = os.environ.get("TRELLO_API_SECRET")
+    TRELLO_BOARD_ID = os.environ.get("TRELLO_BOARD_ID")
+    TRELLO_TOKEN = os.environ.get("TRELLO_TOKEN")
+    TRELLO_USERNAME = os.environ.get("TRELLO_USERNAME")
+
+    def __init__(self):
+        """Base configuration variables."""
+        self.SECRET_KEY = os.environ.get("TRELLO_API_SECRET")

```

Comment:
SECRET_KEY is unrelated to Trello, so it's confusing to use an env var called `TRELLO_API_SECRET`.

It's used by Flask for encrypting a cookie in your browser. Technically it's not used at the moment (since you successfully stopped using session_items), but we will need it again in a later module so I don't mind keeping it around. Just keep the old env var name "SECRET_KEY"